### DISCUSSION TITLE
Support for Pico C/C++ SDK

### DISCUSSION THREAD

**Discussion by pauljurczak** [Bodmer/TFT_eSPI#1887] [Raspberry Pi Pico]
Is there support for Raspberry Pi Pico C/C++ SDK, or I have to use Arduino IDE or PlatformIO with `framework = arduino`?

**Comment by Bodmer** [comment]
The library uses a mix of Arduino library functions and sdk functions, so the Arduino board package is needed. Earle Philhower's RP2040 package is recommended as it is built on the sdk.

**Reply by B3rn475** [reply]
would you accept contributions/refactoring to move the Arduino dependency into a utility that, with build flags, can be swapped with other APIs available in other SDKs?

**Comment by tschesnok** [comment]
Is this still the prefered method to get TFT_eSPI running on a Pico that is running a native C/C++ SDK environment? (I don't want to switch from using VSCode + on hardware debugging). What is the size overhead?

**Comment by tschesnok** [comment]
UPDATE: Never mind. Looks like it fails to build. so there is no good native graphics library for the PICO SDK?

Actually I'm probably looking at this:
https://github.com/fhdm-dev/pico-arduino-compat

Will that build with FTF_eSPI?  I just want to run in the PICO SDK Environment - not Arduino IDE.
